### PR TITLE
[EWPP-1360] EWPP-1366: Add Persons entity reference field storage.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_content_person_reference/oe_content_person_reference.info.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: OpenEuropa Content Persons Field
+description: Allows to reference persons by, for example, providing an entity reference field.
```
Comment (22Alexandra) [inline modules/oe_content_person_reference/oe_content_person_reference.info.yml]:
```suggestion
description: Contains the persons entity reference storage field.
```

File: modules/oe_content_person_reference/oe_content_person_reference.info.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+package: OpenEuropa
+
+type: module
+core_version_requirement: ^8.9 || ^9
+
+dependencies:
+  - drupal:node
+  - drupal:field
```
Comment (22Alexandra) [inline modules/oe_content_person_reference/oe_content_person_reference.info.yml]: We need to add a dependency on oe_content_person submodule.

**Review by 22Alexandra — CHANGES_REQUESTED** [review]
This module should be a submodule of `oe_content_person`.

**Review by 22Alexandra — DISMISSED** [review]

File: modules/oe_content_person/modules/oe_content_person_reference/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# OpenEuropa Persons Field
```
Comment (nagyad) [inline modules/oe_content_person/modules/oe_content_person_reference/README.md]: The module name should be OpenEuropa Person Reference.

File: modules/oe_content_person/modules/oe_content_person_reference/oe_content_person_reference.info.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+name: OpenEuropa Content Persons Field
```
Comment (nagyad) [inline modules/oe_content_person/modules/oe_content_person_reference/oe_content_person_reference.info.yml]: The module name should be OpenEuropa Person Reference.

**Review by nagyad — CHANGES_REQUESTED** [review]

**Review by nagyad — APPROVED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Pull request merged**